BUILD

```diff
@@ -77,11 +77,11 @@ config_setting(
 python_config_settings()
 
 # This should be updated along with build.yaml
-g_stands_for = "guantao"
+g_stands_for = "gallactic"
```

Comment:
Two L's? Does this refer to [the blockchain project](https://github.com/gallactic/gallactic)?

Reply:
That would be interesting, but this was just a subtle typo. Corrected.